hello-world/src/main/java/com/example/hello_world/MainActivity.java

```diff
@@ -0,0 +1,197 @@
+            Request request = new Request.Builder()
+                .url(String.format(URL_CREATE_ACCOUNT, account.getPublicAddress()))
+                .get()
+                .build();
+            okHttpClient.newCall(request)
+                .enqueue(new Callback() {
+                    @Override
+                    public void onFailure(@NonNull Call call, @NonNull IOException e) {
+                        fireOnFailure(callbacks, e);
+                    }
+
+                    @Override
+                    public void onResponse(@NonNull Call call, @NonNull Response response) {
+                        int code = response.code();
+                        response.close();
+                        if (code != 200) {
+                            fireOnFailure(callbacks,
+                                new Exception("Create account - response code is " + response.code()));
+                        }
+
+                        fireOnSuccess(callbacks);
+                    }
+                });
+        }
+    }
+
+    public void getKinBalance(KinAccount account) {
+        kin.sdk.Request<Balance> balanceRequest = account.getBalance();
+        balanceRequest.run(new ResultCallback<Balance>() {
+
+            @Override
+            public void onResult(Balance result) {
+                Log.d(TAG, "The balance is: " + result.value(PRECISION));
+            }
+
+            @Override
+            public void onError(Exception e) {
+                e.printStackTrace();
+            }
+        });
+    }
+
+    public void transferKin(KinAccount sender, String targetPublicAddress, BigDecimal amountInKin) {
+
+        // build the transaction request and run the request asynchronously
+        // the sender is a kin account that transfer Kins to the target public address
+        // each transaction will be charged a fee
+        // memo will state the transaction's reason
+        sender.buildTransaction(targetPublicAddress, amountInKin, FEE, MEMO).run(new ResultCallback<Transaction>() {
+
+            @Override
+            public void onResult(Transaction transaction) {
+                Log.d(TAG, "The transaction id before sending: " + transaction.getId().id());
+
+                sender.sendTransaction(transaction).run(new ResultCallback<TransactionId>() {
+
+                    @Override
+                    public void onResult(TransactionId id) {
+                        Log.d(TAG, "The transaction id: " + id);
+                        getKinBalance(kinAccount);
+                    }
+
+                    @Override
+                    public void onError(Exception e) {
+                        e.printStackTrace();
+                    }
+                });
+            }
+
+            @Override
+            public void onError(Exception e) {
+                e.printStackTrace();
+            }
+
+        });
+    }
+
+    public void addBalanceListeners(KinAccount account) {
+        account.addBalanceListener(
```

Comment:
I think it worth adding remove listeners example (on activity `onStart` `onDestory`)